.env.example

```diff
@@ -3,3 +3,8 @@ FACEBOOK_APP_ID=
 FACEBOOK_APP_SECRET=
 REMOTE_DATABASE_URL=
```

Comment:
oh hmm what is this for?

Reply:
No idea, I couldn't find it mentioned in the codebase -- I guess it was for one of the rake tasks maybe?